### DISCUSSION TITLE
【求助】如何使用github

### DISCUSSION THREAD

**Discussion by silverwolf0000** [share121/inter-knot#3081] [其他]
为了绳网才了解到github，所以想了解一下。求大佬来帮帮我<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/955210989c73f207db2268b43b95f7d5_2639242042836509216.png" alt="阿姬期待" width="75" >

**Comment by x97-1** [comment]
<h4>只是一个存储库而已。git clone 启动copy人生。<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" ></h4>

**Comment by 1ctrl-cv** [comment]
可以去油叉搜一下“github中文化插件”

**Reply by silverwolf0000** [reply]
ok谢谢
